### PR TITLE
Removed VRMLoader

### PR DESCRIPTION
**Pull request by mrdoob**
**Description**

People should use https://github.com/pixiv/three-vrm instead.

/fyi @0b5vr

### CODE REVIEW

File: examples/files.json [hunk 1]
```diff
@@ -120,7 +120,6 @@
 		"webgl_loader_texture_tga",
 		"webgl_loader_ttf",
 		"webgl_loader_vox",
-		"webgl_loader_vrm",
 		"webgl_loader_vrml",
```
Comment (makc) [inline examples/files.json:123]: @mrdoob you missed the spot

Comment (mrdoob) [reply examples/files.json:123]: Hmm?

Comment (makc) [reply examples/files.json:123]: you removed webgl_loader_vrm with the anime girl but not webgl_loader_vrml with the building

Comment (makc) [reply examples/files.json:123]: wait, are these actually two different loaders 🤦

Comment (mrdoob) [reply examples/files.json:123]:
```js
VRMLoader
VRMLLoader
```

Spot the difference 😁

**Pull request merged**